### PR TITLE
Hypershift: add assisted-image-service to base_images

### PR DESCRIPTION
**Pull request by eranco74**

### CODE REVIEW

**Review by eliorerz — APPROVED** [review]

File: ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 base_images:
+  assisted-image-service:
+    name: ocm-2.5
```
Comment (osherdp) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml:3]:
```suggestion
    name: ocm-2.6
```

**Review by osherdp — CHANGES_REQUESTED** [review]

**Comment by osherdp** [comment]
^
/hold

**Review by osherdp — APPROVED** [review]

**Pull request merged**